core/src/stages/compute_ik.cpp

```diff
@@ -49,6 +49,18 @@
 #include <iterator>
 #include <ros/console.h>
 
+namespace {
+bool isIKSolutionValid(const planning_scene::PlanningScene* planning_scene,
+                       const kinematic_constraints::KinematicConstraintSet* constraint_set,
+                       moveit::core::RobotState* state, const moveit::core::JointModelGroup* jmg,
+                       const double* ik_solution) {
+	state->setJointGroupPositions(jmg, ik_solution);
+	state->update();
+	return (!planning_scene || !planning_scene->isStateColliding(*state, jmg->getName())) &&
```

Comment:
Collision checking is redundant to line 399.

Reply:
Thanks